eng/common/pipelines/templates/steps/save-package-properties.yml

```diff
@@ -26,10 +29,12 @@ steps:
       - task: Powershell@2
         displayName: Generate PR Diff
         inputs:
-          filePath: ${{ parameters.ScriptDirectory }}/Generate-PR-Diff.ps1
-          arguments: >
+          targetType: inline
```

Comment:
One of the downsides of using inline scripts is that you don't get to see the command line being called. Can you not pass these via the arguments property still?

Reply:
That's how we were doing it at first, but it has the downside of needing to do more de-serializing and type checking at multiple places in the scripts. It was enough extra annoyance we felt it was simpler to just do it at this level.

Reply:
Apparently not with the converts. @benbp and I had chatted about this.
`-ExcludePaths ('${{ convertToJson(parameters.ExcludePaths) }}' | ConvertFrom-Json)`

Reply:
I believe you could still use that expression as part of the arguments.